crates/weaver_live_check/src/advice.rs

```diff
@@ -125,6 +131,31 @@ impl Advisor for DeprecatedAdvisor {
                 }
                 Ok(advices)
             }
+            SampleRef::Event(sample_event) => {
+                let mut advices = Vec::new();
+                if let Some(group) = registry_group {
+                    if let Some(deprecated) = &group.deprecated() {
+                        advices.push(PolicyFinding {
+                            id: DEPRECATED_ADVICE_TYPE.to_owned(),
+                            context: json!({
+                                EVENT_NAME_ADVICE_CONTEXT_KEY: sample_event.event_name.clone(),
+                                DEPRECATION_REASON_ADVICE_CONTEXT_KEY: deprecated_to_reason(deprecated),
+                                DEPRECATION_NOTE_ADVICE_CONTEXT_KEY: deprecated,
+                            }),
+                            message: format!(
+                                "Event '{}' is deprecated; reason = {}, note = {}",
+                                sample_event.event_name.clone(),
+                                deprecated_to_reason(deprecated),
+                                deprecated
+                            ),
+                            level: FindingLevel::Violation,
+                            signal_type: Some("event".to_owned()),
```

Comment:
I think we should use `log` as a signal (that's the spec terminology). Events are subtype of logs